src/System.Runtime/tests/System/NullableMetadataTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿
+using System.Linq;
+using System.Reflection;
+using Xunit;
+
+namespace System.Runtime.Tests
+{
+    public class NullableMetadataTests
+    {
+        private const string NullableAttributeFullName = "System.Diagnostics.CompilerServices.NullableAttribute";
+        private const string NullableContextAttributeFullName = "System.Diagnostics.CompilerServices.NullableContextAttribute";
+        private const string NullablePublicOnlyAttributeFullName = "System.Diagnostics.CompilerServices.NullablePublicOnlyAttribute";
+
+        [Fact]
+        public static void NullableAttributesOnPublicApiOnly()
+        {
+            MemberInfo[] internalMembers = typeof(string).GetMembers(BindingFlags.NonPublic | BindingFlags.Static | BindingFlags.Instance);
```

Comment:
Can you make this a theory for a few different types across a few different assemblies?  Right now it's only testing corelib.

Reply:
Good idea. I'll update. I also realized I needed to cover more testing, like looking at the return attributes and parameter attributes. Also, my full name types are wrong, `System.Diagnostics.CompilerServices` -> `System.Runtime.CompilerServices` 😆

Reply:
> Also, my full name types are wrong

Issues like that are why I requested the positive test as well :)

> also realized I needed to cover more testing, like looking at the return attributes and parameter attributes

Great, but it doesn't need to be perfect. This is just meant to be a sanity test, to catch the 90% case.